plugins/woocommerce-blocks/assets/js/blocks/mini-cart/quantity-badge/index.tsx

```diff
@@ -48,7 +56,7 @@ const QuantityBadge = ( {
 				className="wc-block-mini-cart__badge"
```

Comment:
Can't we just skip showing this span if it shouldn't display? That's what I was getting at in my previous comments about `QuantityBadge` but I didn't realise this component also showed the icon